src/igvfd/audit/file.py

```diff
@@ -112,3 +112,34 @@ def audit_file_no_file_format_specifications(value, system):
             f'has no `file_format_specifications`.'
         )
         yield AuditFailure(audit_message.get('audit_category', ''), f'{detail} {audit_message.get("audit_description", "")}', level=audit_message.get('audit_level', ''))
+
+
+@audit_checker('IndexFile', frame='object')
+@audit_checker('AlignmentFile', frame='object')
+@audit_checker('TabularFile', frame='object')
+@audit_checker('ReferenceFile', frame='object')
+@audit_checker('ModelFile', frame='object')
+@audit_checker('SequenceFile', frame='object')
+def audit_file_missing_anvil_url(value, system):
+    '''
+    [
+        {
+            "audit_description": "Released and archived files that are restricted access are expected to be loaded in anvil.",
+            "audit_category": "missing anvil url",
```

Comment:
Wondering if there will be cases in future where we move files that aren't controlled access to Anvil.

Reply:
I guess if we introduce another workspace for hosting restricted data I can update this to be more general. The audit would have to be updated anyways coz the property anvil_url is specific to that workspace atm.

Reply:
Do you mean that the property has aregex that specifies the url prefix?

Reply:
There is no regex, but to submit anything other than anvil to anvil_url would be weird and if there's another workspace I think that would necessitate either another property or to rename anvil_url to something more general.

Reply:
There will be several end points at anvil.  Examples
https://anvil.terra.bio/#workspaces/anvil-datastorage/AnVIL_IGVF_GRU
https://anvil.terra.bio/#workspaces/anvil-datastorage/AnVIL_IGVF_GRU-PUB